server/routes/games.js

```diff
@@ -38,10 +46,14 @@ router.route('/multiple/').get( async (req, res) => {
         }
       }
       const games = await Promise.all(gamesArray);
-      res.send(games);
+      if (invalidGameIds.length > 0) {
+        res.status(400).send('Invalid game ID(s), please enter valid ID(s)');
+      } else {
+        res.status(200).send(games);
+      }
     })(multipleGameIds)
```

Comment:
Out of curiosity, why is this function being invoked?

Reply:
I wasn't comfortable was async/await since I haven't used it much and this was what worked for me in my head lol